.github/workflows/wcag_test.yml

```diff
@@ -0,0 +1,24 @@
+name: WCAG tests
```

Comment:
I think we should make an item to add the WCAG testing infra to the common repo.